mmv1/templates/terraform/examples/dialogflowcx_tool_version_connector.tf.tmpl

```diff
@@ -0,0 +1,120 @@
+resource "google_dialogflow_cx_agent" "agent" {
+  provider = google-beta
+  display_name = "{{index $.Vars "agent_name"}}"
+  location = "us-central1"
+  default_language_code = "en"
+  time_zone = "America/New_York"
+  description = "Example description."
+  delete_chat_engine_on_destroy = true
+}
+
+resource "google_integration_connectors_connection" "integration_connector" {
+  provider = google-beta
+  name     = "terraform-df-cx-tool-connection"
```

Comment:
This should be a variable set through the template like `{{index $.Vars "agent_name"}}` up above. This lets us add random suffixes to these to ensure they can be run in parallel.

Same for the dataset & table ids below